### PR TITLE
APEXMALHAR-2504 Allow for customization of TwitterSampleInput ConfigurationBuilder

### PR DESCRIPTION
**Pull request by tweise**
@amberarrow please have a look.

### CODE REVIEW

File: contrib/src/main/java/com/datatorrent/contrib/twitter/TwitterSampleInput.java [hunk 1]
```diff
@@ -204,6 +205,13 @@ public void onException(Exception ex)
     }
   }
 
+  /**
+   * Allow derived classes to customize the configuration
+   */
+  protected void setupConfigurationBuilder(ConfigurationBuilder cb)
+  {
```
Comment (amberarrow) [inline contrib/src/main/java/com/datatorrent/contrib/twitter/TwitterSampleInput.java]: Would be better to move the configuration code here from setup(); derived classes would then have the option to invoke or ignore the superclass method.

**Comment by amberarrow** [comment]
Ready to merge once the comment is addressed.

File: contrib/src/main/java/com/datatorrent/contrib/twitter/TwitterSampleInput.java (hunk 1 above)
Comment (tweise) [reply contrib/src/main/java/com/datatorrent/contrib/twitter/TwitterSampleInput.java]: Agreed, also looks much cleaner.

**Comment by tweise** [comment]
@amberarrow PR updated

**Comment by amberarrow** [comment]
Good to go.

**Pull request merged**